### DISCUSSION TITLE
The info icon usage

### DISCUSSION THREAD

**Discussion by QiiFeng** [twilio-labs/paste#1888] [Q&A] [answered]
Hey folks, 

I am adding a new action for the operator card, which gives users the option to link directly to the Operator Detail Page. This would allow someone to understand the details of an operator and understand what they are looking at. 

I am thinking of using the info icon to implement the requirement above.  Basically when the mouse over the icon, a tooltip appears. Once the icon is clicked, a new page will be opened with a new tab. 

My question is In the Paste is the info icon itself can be used to open a new page when clicking on it? I checked the console, it seems when clicking the info icon, what usually happens is a popover shows up. 

<img width="1706" alt="WeChat71341bbb2c2cb6dba274372927417cce" src="https://user-images.githubusercontent.com/86003634/134706726-371d3999-fa2a-4b1c-9d70-7569cc2b4b32.png">

Let me know your thoughts. 

Thanks 
QI

**Comment by loreina** [comment]
👋  Hey @QiiFeng, I passed this along to the team for discussion and we'll get back to you soon.

**Comment by loreina** [comment, marked as answer]
Hi @QiiFeng, thanks for being patient on this. 

It'd be perfectly acceptable to use the info icon with a tooltip that on click, will redirect users to some sort of information/support page.

**Reply by QiiFeng** [reply]
@loreina Great, thanks
